check_patroni/cluster.py

```diff
@@ -15,24 +15,50 @@ def replace_chars(text: str) -> str:
 
 class ClusterNodeCount(PatroniResource):
     def probe(self) -> Iterable[nagiosplugin.Metric]:
+        def debug_member(member: Any, health: str) -> None:
+            _log.debug(
+                "Node %(node_name)s is %(health)s: role %(role)s state %(state)s.",
+                {
+                    "node_name": member["name"],
+                    "health": health,
+                    "role": member["role"],
+                    "state": member["state"],
+                },
+            )
+
+        # get the cluster info
         item_dict = self.rest_api("cluster")
+
         role_counters: Counter[str] = Counter()
         roles = []
         status_counters: Counter[str] = Counter()
         statuses = []
+        healthy_member = 0
 
         for member in item_dict["members"]:
             roles.append(replace_chars(member["role"]))
             statuses.append(replace_chars(member["state"]))
+
+            if member["role"] in ["leader"] and member["state"] == "running":
+                healthy_member += 1
+                debug_member(member, "healthy")
+                continue
+
+            if member["role"] in ["standby_leader", "replica", "sync_standby"] and (
+                (self.has_detailed_states() and member["state"] in ["streaming"])
+                or (not self.has_detailed_states() and member["state"] in ["running"])
+            ):
```

Comment:
No need to use `in [<a single item>]`:
```suggestion
            if member["role"] == "leader" and member["state"] == "running":
                healthy_member += 1
                debug_member(member, "healthy")
                continue

            if member["role"] in ["standby_leader", "replica", "sync_standby"] and (
                (self.has_detailed_states() and member["state"] == "streaming")
                or (not self.has_detailed_states() and member["state"] == "running")
            ):
```

Also, readability might be improved by setting:
```python
state, role = member["state"], member["role"]
```
upfront at the beginning of the loop.

Reply:
done as proposed, thanks